src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -366,6 +366,15 @@
       <CustomCommands>dotnet $(XUnitRunnerDll) %(XUnitWrapperDlls) $(XUnitRunnerArgs)</CustomCommands>
       <CustomCommands Condition=" '%(TestGroup)' != '' ">dotnet $(XUnitRunnerDll) %(XUnitWrapperDlls) $(XUnitRunnerArgs) -trait TestGroup=%(TestGroup)</CustomCommands>
     </XHarnessApkToTest>
+    
+    <XHarnessAppBundleToTest Include="@(Payloads->Metadata('PayloadZipFile'))" Condition="'$(TargetOS)' == 'iOSSimulator' or '$(TargetOS)' == 'tvOSSimulator'">
+      <Targets Condition="'$(TargetArchitecture)' == 'arm64'">ios-simulator-64</Targets>
+      <Targets Condition="'$(TargetArchitecture)' == 'x64'">ios-simulator-64</Targets>
+      <Targets Condition="'$(TargetArchitecture)' == 'x86'">ios-simulator-32</Targets>
```

Comment:
```suggestion
      <TestTarget Condition="'$(TargetArchitecture)' == 'arm64'">ios-simulator-64</TestTarget>
      <TestTarget Condition="'$(TargetArchitecture)' == 'x64'">ios-simulator-64</TestTarget>
      <TestTarget Condition="'$(TargetArchitecture)' == 'x86'">ios-simulator-32</TestTarget>
```

Reply:
Also, beware the 32 bit simulator hasn't been run like ever. Check with someone that this is needed or even possible to get `x86` here

Reply:
Good catch! I just check the requirements and it is actually not a requirement for runtime tests to run on x86 simulator. Thus, I removed it.